### PR TITLE
Fixes two more runtimes

### PR DESCRIPTION
**Pull request by AnturK**

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/fever.dm [hunk 1]
```diff
@@ -36,6 +36,6 @@ Bonus
 	return
 
 /datum/symptom/fever/proc/Heat(mob/living/M, datum/disease/advance/A)
-	var/get_heat = (sqrt(21+A.totalTransmittable()*2))+(sqrt(20+A.totalStageSpeed()*3))
+	var/get_heat = (sqrt(max(0,21+A.totalTransmittable()*2)))+(sqrt(max(0,20+A.totalStageSpeed()*3)))
```
Comment (duncathan) [inline code/datums/diseases/advance/symptoms/fever.dm]: are you sure the bug here isn't that one of `A.totalTransmittable` or `A.totalStageSpeed` is negative? This looks to me like you're just making it fail silently.
At the very least, it looks like you should make it `max(21,21+A.totalTransmittable()*2)` and `max(20,20+A.totalStageSpeed()*3)`

Comment (AnturK) [inline code/datums/diseases/advance/symptoms/fever.dm]: Hmm, let me check i assumed there's simply some disease that sets these to some negative define

File: code/game/machinery/syndicatebomb.dm [hunk 2]
```diff
@@ -33,7 +33,8 @@
 		STOP_PROCESSING(SSfastprocess, src)
 		detonation_timer = null
 		next_beep = null
-		countdown.stop()
+		if(countdown)
+			countdown.stop()
```
Comment (phil235) [inline code/game/machinery/syndicatebomb.dm]: that's just hiding the problem. I'm fixing the actual bug in #21309

Comment (AnturK) [inline code/game/machinery/syndicatebomb.dm]: Oh, it was explosions. Thought countdowns were parametrized. I'll drop this one

File: code/datums/diseases/advance/symptoms/fever.dm [hunk 3]
```diff
@@ -36,6 +36,6 @@ Bonus
 	return
 
 /datum/symptom/fever/proc/Heat(mob/living/M, datum/disease/advance/A)
-	var/get_heat = (sqrt(21+A.totalTransmittable()*2))+(sqrt(20+A.totalStageSpeed()*3))
+	var/get_heat = (sqrt(max(21,21+A.totalTransmittable()*2)))+(sqrt(max(21,20+A.totalStageSpeed()*3)))
```
Comment (phil235) [inline code/datums/diseases/advance/symptoms/fever.dm]: why not `max(0, 21+A.totalTransmittable()*2)`?

Comment (AnturK) [inline code/datums/diseases/advance/symptoms/fever.dm]: So the result is actually similar to the minimal value before the negative values are added. Honestly this whole line should be replaced with readable clamp..

**Pull request merged**